posts/2018-02-09-serverless-and-graphql-in-love.md

```diff
@@ -0,0 +1,540 @@
+---
+title: Serverless and GraphQL : The best of both worlds  
+description: Welcome to the first of a series of technical blog posts on Serverless and GraphQL. In this blog, we will learn how to create GraphQL endpoints using Serverless Framework, Lambda, DynamoDB, RDS and a REST backend.   
+date: 2018-02-13  
+layout: Post  
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/graphql.jpeg  
+authors:  
+  - SiddharthGupta
+---
+
+![alt text](https://user-images.githubusercontent.com/1587005/36030132-d45feefe-0d5a-11e8-986c-92dad62f3408.png "Serverless and GraphQL family")
+
+## Introduction
+
+Over the last four years, I've been exploring the world of big data, building real-time and batch systems at scale. For the last couple of months, I've been building products with serverless architectures here at Glassdoor.
+
+Given the intersection of serverless and big data, there have been a few questions on everyone's mind: 
+
+1) How can we build low latency APIs to server these complex, high dimensional and big datasets which can scale on demand?
+2) Can we fire single query which can construct a nested response from multiple data sources?
+3) Is there an easy way to secure the endpoint, paginate through the data, aggregate the results at scale and what not? 
+4) Can we pay per query execution rather than running fleet of servers?
+
+The answer for us ended up largely being GraphQL.
+
+This blog aims to show you how you, too, can streamline your existing workflow and handle complexity with ease. While I won't be digging deep into specific things Glassdoor was working on, I *will* be showing you a pretty related example that utilizes a mini Twitter clone I made.
+
+In this post, I’ll cover how to create Serverless GraphQL endpoints using DynamoDB and RDS, which we'll wrap around a REST API.
+
+Ready? Awesome. Let's go.
+
+## What is GraphQL?
```

Comment:
not sure if emojis are being rendered properly. 
<img width="862" alt="screen shot 2018-02-12 at 9 17 36 am" src="https://user-images.githubusercontent.com/1587005/36109892-9781ba26-0fd5-11e8-8541-e532f18ad6d7.png">